### DISCUSSION TITLE
May I ask how to reverse query the Geometry object through the Layer object!

### DISCUSSION THREAD

**Discussion by lucinelouhi** [Mapsui/Mapsui#2618] [Q&A]
May I ask how to reverse query the Geometry object through the Layer object! I need to query the Geometry objects in the layer and perform operations on another Geometry.

**Comment by pauldendulk** [comment]
I think you should not try to do this through Mapsui. Mapsui is for rendering, NTS is for geometry operations. You could fully manage you NTS geometries outside of Mapsui and implement an IProvider to render those geometries in Mapsui.
